Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -61,8 +62,15 @@ public struct Conv1D<Scalar: TensorFlowFloatingPoint>: Layer {
 
     /// Returns the output obtained from applying the layer to the given input.
     ///
-    /// - Parameter input: The input to the layer `[batchCount, width, inputChannels]`.
-    /// - Returns: The output `[batchCount, newWidth, outputChannels]`.
+    /// - Parameter input: The input to the layer [batch count, input width, input channel count].
+    /// - Returns: The output of shape [batch count, output width, output channel count],
```

Comment:
The return description seems more like a summary of this operation, so I think it should be moved to the summary. Same for other such return descriptions. Returns descriptions are usually a simple noun phrase ending with a period.

Reply:
Hey @rxwei. Is it okay if I keep breaks in between computations of the output dimensions like those in the comment string below?

/// The output width is computed as:                                                                        
///                                                                          
/// output width =                                   
/// (input width + 2 * padding size - (dilation * (filter width - 1) + 1)) / stride + 1                        
///                              
/// and padding size is determined by the padding scheme. Note that padding size equals zero                   
/// when using `.valid`.

Reply:
Yep, I think it is okay.
I would recommend turning the `Note ...` sentence into a `- Note: Padding size equals ...` documentation item.